### PR TITLE
Switch to yarn

### PR DESCRIPTION
**Pull request by McInkay**

### CODE REVIEW

**Comment by McInkay** [comment]
(This one is really easy to approve...)

**Comment by McInkay** [comment]
RIP, @TheSonAlsoRises 's review doesn't count :-P

**Comment by McInkay** [comment]
Ok, I have updated this to include installation instructions and include the commands for both yarn and npm. I think this is what we agreed in chat @SnowPhoenix 

I have also removed the stuff for pre-6, because that's no longer LTS so there's no reason to support it anymore.

File: README.md [hunk 1]
```diff
@@ -10,17 +10,24 @@ An IRC bot created by the /r/pokemontrades team for our use.
 * Manipulating reddit toolbox usernotes
 * Shortcuts
 
+## Pre-requisites
+
+* Node 6+ ([node installation instructions](https://nodejs.org/en/download/package-manage))
```
Comment (robdy) [inline README.md]: `https://nodejs.org/en/download/package-manager` (missing `r`)

Comment (McInkay) [reply README.md]: rip.

Fixed.

**Pull request merged**